kalendar-foundation/src/commonMain/kotlin/com/himanshoe/kalendar/foundation/action/OnDayClick.kt

```diff
@@ -16,11 +16,11 @@
 
 package com.himanshoe.kalendar.foundation.action
 
-import com.himanshoe.kalendar.foundation.event.KalendarEvent
+import com.himanshoe.kalendar.foundation.event.KalenderEvent
 import kotlinx.datetime.LocalDate
 
 fun LocalDate.onDayClick(
-    events: List<KalendarEvent>,
+    events: List<KalenderEvent>,
```

Comment:
from performance perspective, can you migrate all list usages to ImmutableList?

Reply:
I can, but that all plan is for later implementation!